app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
```

Comment:
- [ ] Good job adding the view files team, but it's expected the application allows for registering a new user and allowing login for a registered user. Currently, I cannot access the route for the login and sign-up pages. Please make sure these features are in place and we can register and log in as a user.